patches/tModLoader/Terraria/ModLoader/Core/ModCompile.cs

```diff
@@ -814,3 +782,4 @@ public override AssemblyDefinition Resolve(AssemblyNameReference name)
 		}
 	}
 }
+#endif
```

Comment:
an accidental reformat, roll this back please. It's important to preserve the change history of this class.